sdks/uniswapx-sdk/src/builder/V3DutchOrderBuilder.ts

```diff
@@ -325,13 +325,20 @@ export class V3DutchOrderBuilder extends OrderBuilder {
     relativeAmounts: bigint[]
   ): BigNumber {
     if (relativeAmounts.length == 0) {
-      throw new Error("relativeAmounts cannot be empty");
+      return startAmount;
     }
+
     // Find the minimum of the relative amounts
     const minRelativeAmount = relativeAmounts.reduce(
       (min, amount) => (amount < min ? amount : min),
       relativeAmounts[0]
```

Comment:
it might be cleaner to set this initial value as `0`  and then you can remove lines 336-341 (since subtracting 0 will give you the startAmount).

Reply:
Fixed